packages/mercury/docs/installation-for-angular.md

```diff
@@ -0,0 +1,372 @@
+<img src="./images/mercury-brand.svg" alt="Mercury brand" width="30" />
+
+Mercury Design System
+
+# <img src="./images/angular.svg" alt="Angular isotype" width="30" /> Installation for Angular
+
+- [1. Installation](#1-installation)
+
+- [2. Usage](#2-usage)
+
+  - [2.1. Before starting, define the path for the CSS bundles, custom fonts and icons](#21-before-starting-define-the-path-for-the-css-bundles-custom-fonts-and-icons)
+
+  - [2.2. Create the CSS bundles](#22-create-the-css-bundles)
+
+    - [2.2.1. Using the CLI to create the CSS bundles](#221-using-the-cli-to-create-the-css-bundles)
+
+    - [2.2.2. Using SASS to transpile the CSS bundles](#222-using-sass-to-transpile-the-css-bundles)
+
+    - [2.2.3. Using already generated CSS bundles](#223-using-already-generated-css-bundles)
+
+  - [2.3. Copy the CSS bundles, custom fonts and icons to the dev and prod builds](#23-copy-the-css-bundles-custom-fonts-and-icons-to-the-dev-and-prod-builds)
+
+    - [2.3.1. Copying assets](#231-copying-assets)
+
+  - [2.4. Register Mercury and Chameleon utilities](#24-register-mercury-and-chameleon-utilities)
+
+  - [2.5. Style the base application](#25-style-the-base-application)
+
+    - [2.5.1. Adding base styles using normal HTML](#251-adding-base-styles-using-normal-html)
+
+  - [2.6. Style the components with the CSS bundles](#26-style-the-components-with-the-css-bundles)
+
+    - [2.6.1 How to style a component](#261-how-to-style-a-component)
+
+  - [2.7. Download the icon paths](#27-download-the-icon-paths)
+
+  - [2.8. Set the dark and light mode](#28-set-the-darklight-mode)
+
+- [3. CSS bundles mapping](#3-css-bundles-mapping)
+
+- [4. CLI flags](#4-cli-flags)
+
+## 1. Installation
+
+```bash
+npm i @genexus/mercury
+```
```

Comment:
I think we can move this section to the main readme file. We should also explain how to install Chameleon too.

```bash
npm i --save @genexus/chameleon-controls-library @genexus/mercury
```